pkg/loader/compose/utils.go

```diff
@@ -33,6 +33,9 @@ const (
 	LabelServiceExpose = "kompose.service.expose"
 	// LabelServiceExposeTLSSecret  provides the name of the TLS secret to use with the Kubernetes ingress controller
 	LabelServiceExposeTLSSecret = "kompose.service.expose.tls-secret"
+
+	// ServiceTypeHeadless...
+	ServiceTypeHeadless = "Headless"
```

Comment:
most other variables are lowercase, should we use `headless`  instead?

Reply:
This is a special case. Since `api.ServiceTypeNodePort/api.ServiceTypeClusterIP...` use `NodePort` `LoadBalancer`... So i think my be follow there rules may be better

Reply:
This is a special case. Since kubernetes api does not have a "Headless' type, it has `api.ServiceTypeClusterIP=ClusterIP`.... This is var is only for internal use, others are custom labels that will show in the compose file